.github/workflows/migration.yml

```diff
@@ -34,13 +34,15 @@ jobs:
       with:
         node-version: '12'
 
-    # Setup gcloud CLI
-    - name: Setup gcloud CLI
-      uses: google-github-actions/setup-gcloud@v0
-      with:
-        service_account_email: ${{ secrets.GOOGLE_SERVICE_ACCOUNT_EMAIL }}
-        service_account_key: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
-        export_default_credentials: true
+    - id: 'auth'
```

Comment:
switching over to using identity provider as done in ci.yml

https://github.com/getsentry/eng-pipes/blob/04f8a2adeb0de29f3a7f0d0675cd4be91dbef55e/.github/workflows/ci.yml#L125